modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -112,6 +112,19 @@ function editIssue()
     updateHistory($historyValues, $issueID);
     updateComments($_POST['comment'], $issueID);
 
+    // Attachment for new issue.
+    if (isset($_FILES['file'])) {
+        $attachment = new \LORIS\issue_tracker\UploadHelper();
+        $attachment->setupUploading(
+            $user,
+            $_FILES,
+            array(
+                'fileDescription' => '',
```

Comment:
Shouldn't this be taken from the form instead of set to a blank string?

Reply:
@johnsaigle when first creating an issue there isn't the field for a fileDescription. The choice was made that information would be added in the comment field. Once the issue is made the option of adding file descriptions is then an option when uploading a file.